embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/config/store/RemoteConfigStoreImpl.kt

```diff
@@ -42,7 +40,7 @@ internal class RemoteConfigStoreImpl(
 
     override fun saveResponse(response: ConfigHttpResponse) {
         try {
-            GZIPOutputStream(configFile.outputStream().buffered()).use { stream ->
+            configFile.outputStream().buffered().use { stream ->
```

Comment:
The data coming back from the server is not compressed, or is automatically ungzipped?

Reply:
The data is ungzipped in `OkHttpRemoteConfigSource`